prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/mailTemplates/bodyTemplate.header.html

```diff
@@ -30,10 +30,10 @@
         max-width: 45% !important;
       }
       .note {
-        opacity: .6;
+        color: #c6934b;
         font-size: small;
-        border: 1px solid #ddd;
-        background: #f5f5f5;
+        border: 1px solid #c6934b;
+        background: #c6934b10;
```

Comment:
This is quite new, not yet in a stable standard, so I'd rather use:

```suggestion
        background: rgba(198, 147, 75, .065);
```